amazon-sqs/5.6/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,69 @@
+
+The AWS SDK for Java provides a Java API for AWS infrastructure services. Amazon SQS Connector is built using the SDK for Java.
+
+For compatibility information and fixed issues, see the xref:release-notes::connector/amazon-sqs-connector-release-notes-mule-4.adoc[Amazon SQS Connector Release Notes].
+
+== Cluster Mode
+
+Amazon SQS Connector supports listening for messages on the primary node only or on all nodes when running in cluster mode. For information, see xref:amazon-sqs-connector-config-topics.adoc#node-behavior[Receiving Messages in Cluster Mode].
+
+== Queue Support
+
+Amazon SQS Connector provides support for two types of queues:
+
+* Standard Queues
++
+Standard queues have high throughput. However, using these queues can occasionally result in having duplicate copies of messages delivered out of order.
++
+* FIFO Queues
++
+FIFO (First-In-First-Out) queues deliver messages in the order in which they are sent. However, these queues have limited throughput.
+
+For information about creating FIFO queues, see xref:amazon-sqs-connector-config-topics.adoc[Additional Configuration Topics].
+
+== Prerequisites
+
+Before creating an app, you must:
+
+* Have access to the Amazon SQS target resource and Anypoint Platform.
+* Understand how to create a Mule app using Design Center or Anypoint Studio.
+* Have AWS Identity and Access Management (IAM) credentials.
+
+== Audience
+
+Read the Amazon SQS Connector documentation that meets the needs of the type of user that you are:
+
+* Starting user
+** To create your Mule app:
+see xref:amazon-sqs-connector-design-center.adoc[Design Center Configuration]
+or xref:amazon-sqs-connector-studio.adoc[Studio Configuration]. The xref:amazon-sqs-connector-config-topics.adoc[Additional Configuration Information]
+topic helps extend your understanding of Amazon SQS Connector
+** The xref:amazon-sqs-connector-examples.adoc[Examples] topic provides one or more use cases for using the connector.
+* Power user
+** Read the xref:amazon-sqs-connector-xml-maven.adoc[XML and Maven Support],
+xref:amazon-sqs-connector-config-topics.adoc[Additional Configuration Information],
+and xref:amazon-sqs-connector-examples.adoc[Examples] topics.
+
+== Common Use Cases for the Connector
+
+Common use cases for Amazon SQS Connector include sending messages to a queue and retrieving messages from a queue. For an example of these use cases, see xref:amazon-sqs-connector-examples.adoc#install-connector[Amazon SQS Examples].
+
+== Next Step
+
+After you have met the prerequisites and experimented with examples, you are ready to create an app with
+xref:amazon-sqs-connector-design-center.adoc[Design Center] or xref:amazon-sqs-connector-studio.adoc[Anypoint Studio].
```

Comment:
I don't see the Design Center topic in the nav. I may have missed it. Just make sure to remove links to the Design Center topic if we removed that topic.